typings/index.d.ts

```diff
@@ -1200,6 +1202,14 @@ declare module "noblox.js" {
 
     function updateDeveloperProduct(universeId: number, productId: number, name: string, priceInRobux: number, description?: string, jar?: CookieJar): Promise<DeveloperProductUpdateResult>;
 
+    /**
+     * Configures a gamepass with the id `gamepassId` to have a `name` and `description`, and sets the `price` in Robux. If both `name` and `description` are empty, only `price` is changed. Setting `price` to false, 0, or a negative value will place the gamepass off-sale.
+     * Returns `gamepassId` with the changed attributes.
+     * 
+     * NOTE: Updating `name` will affect `description`: you must 'refresh' `description` with each `name` update, or `description` will be cleared.
+     */
+    function configureGamePass(gamepassId: number, name: string, description?: string, price?: number | boolean, jar?: CookieJar): Promise<GamepassInfo>;
```

Comment:
GamepassInfo isn't a valid interface. You'll want to include it towards the top of the file and in the other typings file.

Reply:
Added as `GamePassResponse` interface; seems I missed this one when manually transposing my edits from my working bot directory to the cloned repo- just first pull request things I guess 🙄